confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/RegionGroup.java

```diff
@@ -35,18 +35,20 @@ public class RegionGroup {
 
   private final TRegionReplicaSet replicaSet;
 
-  // For DataRegion, each SeriesSlot * TimeSlot form a slot,
-  // for SchemaRegion, each SeriesSlot is a slot
-  private final AtomicLong slotCount;
+  // For DataRegion, each SeriesSlot and each TimeSlot it contains can form a slot.
+  // Eg: Have SeriesSlot-1, SeriesSlot-1 has TimeSlot-1, TimeSlot-2, Timeslot-3,
+  // then (SeriesSlot1 -> TimeSlot1) is a slot.
+  // For SchemaRegion, each SeriesSlot is a slot
```

Comment:
// For DataRegion, a SeriesSlot and a TimeSlot constitute a DataPartition.
// Eg: A DataRegion contains SeriesSlot-1 which has TimeSlot-1, TimeSlot-2 and Timeslot-3,
 // then (SeriesSlot-1 -> TimeSlot-1) constitute a DataPartition.
 // For SchemaRegion, each SeriesSlot constitute a SchemaPartition.